### DISCUSSION TITLE
Modify sdcard image size from 16GB -> 2.3GB

### DISCUSSION THREAD

**Discussion by EasyIP2023** [OE4T/meta-tegra#641] [Q&A] [answered]
Basically I have a custom layer with a .conf file that changes these two variables

```
ROOTFSPART_SIZE ?= "2000050000"
TEGRAFLASH_SDCARD_SIZE ?= "2200M"
```
When I generate an sdcard image via ./dosdcard.sh this works fine on branch dunfell-32.4.3. I'm able to boot into image and mount on local machine with no problems, but on branch dunfell-32.5.0 when I select machine=jetson-nano-2gb-devkit and do the same thing. It resizes the sdcard image, but I can't boot into it or mount it on local machine. When I select machine=jetson-xavier-nx-devkit it doesn't create the image at all.  Just wondering if there's somewhere else on branch dunfell-32.5.0 that allows me to modify the size of the sdcard image?

This is how far it gets to on machine=jetson-xavier-nx-devkit before stopping itself.
```
[   3.0381 ] Copying signed file in /path/to/signed
[   3.1070 ] tegraparser_v2 --pt flash.xml.bin --generateflashindex /path/to/signed/flash.xml.tmp flash.idx
[   3.5216 ] 
Init...kernel...kernel_b...kernel-dtb...kernel-dtb_b...recovery...recovery-dtb...kernel-bootctrl...kernel-bootctrl_b...RECROOTFS...UDA...APP...
```

**Comment by madisongh** [comment, marked as answer]
IIRC, the flashing tools in L4T R32.5.0 pay more attention to the `num_sectors` attribute in the `<device>` elements of the flash layout XML file.  A mismatch between that file and the actual number of sectors on the card could potentially lead to booting problems, so you might need to use a customized layout with the correct sector count.
